src/main/java/jenkins/plugins/slack/SlackNotifier.java

```diff
@@ -602,6 +628,20 @@ public void setBotUser(boolean botUser) {
             this.botUser = botUser;
         }
 
+        public String getIconEmoji() { return iconEmoji; }
+
+        @DataBoundSetter
+        public void setIconEmoji(String iconEmoji) {
```

Comment:
Something is fishy when you need to add the setter/getter twice.

Surely we could use the same object/class.

Reply:
Aren't these needed to get/set values on configuration pages?

Reply:
This could be done as a separate improvement, but there is definitely some need to reduce the repetition :sweat: